core/connectionlistmodel.cpp

```diff
@@ -7,29 +7,15 @@ ConnectionListModel::ConnectionListModel(QObject *parent) :
 {
 }
 
-QHash <int, QByteArray> ConnectionListModel::roleNames() const
```

Comment:
Seems like something that might be used by QML, but a quick grep did not reveal anything accessing "address". Except "btPairedDevice", but that's something different. Can probably be removed.